src/main/java/net/kaczmarzyk/spring/data/jpa/web/WebRequestProcessingContext.java

```diff
@@ -64,9 +67,14 @@ public QueryContext queryContext() {
 	}
 
 	public String getPathVariableValue(String pathVariableName) {
-		if(resolvedPathVariables == null) {
+		if (resolvedPathVariables == null) {
+			resolvedPathVariables = (Map) webRequest.getAttribute(HandlerMapping.URI_TEMPLATE_VARIABLES_ATTRIBUTE, WebRequest.SCOPE_REQUEST);
+		}
+		//if first method returns null then try alternative path variables resolving method
```

Comment:
Consider adding more descriptive javaodc:

```
//When the request attributes dont not contain information about extracted path variables, 
the path variables are extracted using a fallback approach (which has some limitations, eg. it does not support global prefixes). 
```

Reply:
added examples and more to class description

Reply:
see further comments